server/routes/auth/index.js

```diff
@@ -26,6 +26,9 @@ router.post("/register", async (req, res, next) => {
       process.env.SESSION_SECRET,
       { expiresIn: 86400 }
     );
+
+    res.cookie("token", token, { httpOnly: true });
```

Comment:
You can add the `expires` option, also create `cookieOptions` object and reuse it.